src/Tokenizers/MySQL/ColumnTokenizer.php

```diff
@@ -40,7 +43,11 @@ public function tokenize(): self
 
         $this->consumeGenerated();
 
-        if ($this->columnDataType == 'timestamp' || $this->columnDataType == 'datetime') {
+        if ($this->columnDataType == 'timestamp') {
+            $this->consumeTimestamp();
+        }
```

Comment:
Why does this remove the datetime check? And why are the separated?

Reply:
Shoot, that was a mistake.  I think I checked out the wrong branch or something.